### ISSUE TITLE
Support GitLab as provider

### ISSUE THREAD

**Issue by Al2Klimov** [RISE-GmbH/icingaweb2-module-oidc#2]
I had to to this

```patch
--- a/application/controllers/AuthenticationController.php
+++ b/application/controllers/AuthenticationController.php
@@ -69,3 +69,3 @@ class AuthenticationController extends \Icinga\Controllers\AuthenticationControl
             // Initiate the login process at the OP
-            $oidc->addScope(['profile', 'groups','email']);
+            $oidc->addScope(['profile', 'email']);
             $redirect="dashboard";
```

to make it working. But I'm pretty sure this is not what you want.

Also, I had to specify the non-obvious redirect URL https://minigame.prod.aklimov.net-dump.de/icingaweb2/oidc/authentication/realm in GitLab.

**Comment by moreamazingnick** [comment]
yes the redirection url would be https://minigame.prod.aklimov.net-dump.de/icingaweb2/oidc/authentication/realm?name=<yourprovidername>
I will add that to the documentation.

For the github issue, I can make the groups claim optional...

Best Regards
Nicolas

**Comment by moreamazingnick** [comment]
can you try this release https://github.com/RISE-GmbH/icingaweb2-module-oidc/releases/tag/v0.6.2 and turn on the nooidcgroups switch for the gitlab provider?
